src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from openzeppelin.access.ownable.library import Ownable
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin, SignatureBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+// Local dependencies
+from kakarot.accounts.library import GenericAccount, Account_kakarot_address, Account_evm_address
+from starkware.starknet.common.syscalls import get_tx_info, get_caller_address, replace_class
+from starkware.cairo.common.math import assert_le, unsigned_div_rem
+from starkware.cairo.common.alloc import alloc
+
+@constructor
+func constructor{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}(kakarot_address: felt, evm_address: felt) {
+    Account_kakarot_address.write(kakarot_address);
+    Account_evm_address.write(evm_address);
+    return ();
+}
+
+// @title EVM smart contract account representation.
+
```

Comment:
remove line break

Reply:
up

Reply:
I think the line break between the title of the contract and the first @notice of the first function should stay. More aesthetic